Omex.sln

```diff
@@ -336,6 +340,22 @@ Global
 		{CDAA3EBA-02AE-41C6-B915-643A69A1BE57}.Release|Any CPU.Build.0 = Release|Any CPU
 		{CDAA3EBA-02AE-41C6-B915-643A69A1BE57}.Release|x64.ActiveCfg = Release|Any CPU
 		{CDAA3EBA-02AE-41C6-B915-643A69A1BE57}.Release|x64.Build.0 = Release|Any CPU
+		{C7C334F5-6A26-4779-B02F-1EA0B1A02799}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{C7C334F5-6A26-4779-B02F-1EA0B1A02799}.Debug|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
Do we still support any cpu?

Reply:
For general libraries yes, we are limited to x64 for libraries that need SF pacakges